worker/workercontroller.go

```diff
@@ -10,16 +10,17 @@ import (
 // Controller holds worker instances.
 // Currently, only local workers are supported.
 type Controller struct {
-	mu sync.Mutex
```

Comment:
Don't we need mu for defaultID?

Reply:
I guess this code needs changes anyway when workers can actually appear dynamically. The current code expects that the default worker is always initialized on startup.